lib/matplotlib/widgets.py

```diff
@@ -974,40 +973,48 @@ class RadioButtons(AxesWidget):
     """
     A GUI neutral radio button.
 
-    For the buttons to remain responsive
-    you must keep a reference to this object.
+    For the buttons to remain responsive you must keep a reference to this
+    object.
 
-    The following attributes are exposed:
+    Connect to the RadioButtons with the :meth:`on_clicked` method.
 
-     *ax*
-        The :class:`matplotlib.axes.Axes` instance the buttons are in
 
-     *activecolor*
-        The color of the button when clicked
+    Attributes
+    ----------
 
-     *labels*
-        A list of :class:`matplotlib.text.Text` instances
+    ax
+        The containing `~.axes.Axes` instance.
+
```

Comment:
can probably do without the blank lines for explanations this short.  same comment below for the Parameters table.